components/FeaturedSection.js

```diff
@@ -0,0 +1,90 @@
+import { useContext, useState } from "react";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Carousel from "react-bootstrap/Carousel";
+import { useMediaQuery } from "react-responsive";
+import { IoIosArrowDropleft, IoIosArrowDropright } from "react-icons/io";
+import IssuesContext from "../contexts/IssuesContext";
+import IndexIssueListItem from "./shared/IndexIssueListItem";
+
+export default function FeaturedSection() {
+  // Set state for next/previous arrows so they disappear on slide
+  const [showArrows, setShowArrows] = useState(true);
+
+  const { issues } = useContext(IssuesContext);
+  const featuredIssues = issues.filter((issue) => issue.isFeatured);
+
+  const isTabletOrMobile = useMediaQuery({ query: "(max-width: 996px)" });
+
+  return (
+    <Row>
+      {/* Render rows with featured cards if tablet or mobile else render carousel */}
+      {isTabletOrMobile ? (
+        <Col className="mx-5">
+          <Row className="pt-3 pb-4 text-primary text-center">
+            <Col>
+              <h4 className="fw-bold">Featured</h4>
+            </Col>
+          </Row>
+          <Row>
+            {/* This slice is incase there are more than 8 issues with the featured label */}
+            {featuredIssues.slice(0, 8).map((issue) => {
+              return (
+                <Col s={12} md={6} key={issue.number} className="px-4 pb-5">
+                  <IndexIssueListItem issue={issue} />
+                </Col>
+              );
+            })}
+          </Row>
+        </Col>
+      ) : (
+        <Col className="px-0">
```

Comment:
i'm seeing a horizontal scrollbar when i'm looking at this on desktop / `lg`. i think nixing this `px-0` should clear it up?

<img width="443" alt="Screen Shot 2022-11-04 at 12 56 51 PM" src="https://user-images.githubusercontent.com/14793120/200033011-1eff9c85-0612-4360-bf93-05056a0c979b.png">

Reply:
Huh weird I dont get that, but nixing it bc it doesn't do much!